test/maybe_.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var throws = require('assert').throws;
+
+var R = require('ramda');
+
+var S = require('..');
+
+var eq = require('./internal/eq');
+var errorEq = require('./internal/errorEq');
+
+describe('maybe_', function() {
```

Comment:
I'm sorry I was not clearer here. This is the [PEP 8][1] -inspired format I like to use:

```javascript
'use strict';

//  Standard library imports, in alphabetical order

//  Third-party imports, in alphabetical order

//  Relative imports, in alphabetical order


//  Code starts here
```

In this case there's actually a fourth import group for the test helpers:

```javascript
'use strict';

var throws = require('assert').throws;

var R = require('ramda');

var S = require('..');

var eq = require('./internal/eq');
var errorEq = require('./internal/errorEq');


describe('maybe_', function() {
//  ...
});
```

I realize many projects don't follow this system (or any other system), but I find this hierarchical approach useful.


[1]: https://www.python.org/dev/peps/pep-0008/